### PR TITLE
VVBykov HW-7 

### PR DESCRIPTION
**Pull request by VladimirVBykov**
The home assignment #7 has been done.

### CODE REVIEW

File: users/vvbykov/exercise/hw7/hw-7-part-2.ipynb
Comment (isergy) [inline users/vvbykov/exercise/hw7/hw-7-part-2.ipynb:1]: ок норм

File: users/vvbykov/exercise/hw7/hw-7-part-check-yourself.ipynb
Comment (isergy) [inline users/vvbykov/exercise/hw7/hw-7-part-check-yourself.ipynb:1]: ок норм

File: users/vvbykov/exercise/hw7/hw-7-part-1.ipynb
Comment (isergy) [inline users/vvbykov/exercise/hw7/hw-7-part-1.ipynb:1]: ок норм

**Review by isergy — APPROVED** [review]
класс!

**Review by theRate — APPROVED** [review]
ok

**Pull request merged**